### DISCUSSION TITLE
How to deal with missing ressources

### DISCUSSION THREAD

**Discussion by tuxpeople** [fluxcd/flux2#2104] [Q&A]
Hi

Does Flux have something like ArgoCD's annotation `argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true`? Or is there another way to work around errors about non existing resources? Example: If I want to add a CRD and a CR in the same deployment, the CR fails until the CRD is installed.

Right now, I work around that with "different stages" which are built upon another. But that's a bit messy in my opinion.

Br
Thomas

**Comment by stefanprodan** [comment]
> I want to add a CRD and a CR in the same deployment, the CR fails until the CRD is installed

I guess you are using an older version of Flux. Starting with v0.18.0 Flux knows how reconcile CRD+CR. See here the upgrade guide: https://github.com/fluxcd/flux2/discussions/1916
